### ISSUE TITLE
Website home image is showing the names of players

### ISSUE THREAD

**Issue by jxl8437** [OvercastNetwork/Issues#1894]
This only applies for one image though (as far as I can tell).

![](http://i.imgur.com/mXHMWew.png)

**Comment by nixter1029** [comment]
I don't even see how this even comes close to a real issue. It has been like this for a while...

**Comment by jxl8437** [comment]
Well, it is different from the others. If none of the rest have name tags, then this one in theory shoudn't

**Comment by Tapetepato** [comment]
i think it is because when you press f1 on 1.8, you still see the nametags

**Comment by Brottweiler** [comment]
Yeah, ideally none of the images should have nametags. But as said, I think it's a 1.8 bug that makes nametags show when you hide the HUD. Not sure if fixed in 1.8.7+.

**Comment by Ipskie** [comment]
https://bugs.mojang.com/browse/MC-56533 not yet fixed
